aws/service.go

```diff
@@ -1027,6 +1027,18 @@ func S3ControlClient(ctx context.Context, d *plugin.QueryData, region string) (*
 	return s3control.NewFromConfig(*cfg), nil
 }
 
+// All requests to create or maintain Multi-Region Access Points are routed to the US West (Oregon) Region.
+// This is true regardless of which Region you are in when making the request, or what Regions the Multi-Region Access Point supports.
+// https://docs.aws.amazon.com/AmazonS3/latest/userguide/ManagingMultiRegionAccessPoints.html
+// S3 multi-region access point supports in China but not in US Gov or US ISO
+func S3ControlMultiRegionAccessClient(ctx context.Context, d *plugin.QueryData) (*s3control.Client, error) {
```

Comment:
Is there an endpoint list available for those particular resource type in the S3 endpoints list in SDK v1? In China, do you know what region it's supported in? Also, is this supported in US GovCloud?

Do you have any other docs or reference mentioning support in non-commercial regions?

Reply:
We use `s3control` package to create clients for multi-region access points. But in the doc, it is mentioned if we are doing any operation on the multi-region access points, all requests to create or maintain Multi-Region Access Points are routed to the US West (Oregon) Region. This is true regardless of which Region you are in when making the request or what Regions the Multi-Region Access Point supports.

[Commercial regions](https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiRegionAccessPointRestrictions.html) which are supporting multi-region access point

The multi-region Access point supports in [china regions](https://www.amazonaws.cn/en/new/2020/amazon-s3-access-points-now-support-copy-api/#:~:text=With%20support%20of%20the%20COPY,support%20these%20S3%20object%20operations), Also checked supported region by passing `cn-northwest-1` region to its service client, the table returned zero rows instead of any endpoint error.

But I didn't find any doc which says it is supported in US GovCloud, so I logged in to the AWS Gov Cloud account which we have today, and I saw there is no such resource type available in any of the regions. Also, I tried to create a Service Client by passing the Gov cloud regions. I m getting the no such host error for that endpoint.
```
 https response error StatusCode: 0, RequestID: , HostID: , request send failed, Get "https://632902152528.s3-control.us-iso-east-1.c2s.ic.gov/v20180820/mrap/instances?maxResults=100": dial tcp: lookup 632902152528.s3-control.us-iso-east-1.c2s.ic.gov: no such host
```